docs/iris/example_code/General/coriolis_plot.py

```diff
@@ -0,0 +1,80 @@
+"""
+Deriving the Coriolis frequency over the globe
+==============================================
+
+This code computes Coriolis frequency and stores them in a cube with
```

Comment:
This code computes Coriolis frequency and stores them in a cube with
This code computes **the** Coriolis frequency and stores ~~them~~ **it** in a cube with